Ecosia/Core/Environment/Environment.swift

```diff
@@ -0,0 +1,33 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+
+public enum Environment: Equatable {
+    case production
+    case staging
+}
+
+extension Environment {
+
+    public static var current: Environment {
+        #if PRODUCTION
```

Comment:
Wasn't this part of Core logic? Should we use MOZ flags here now?

Reply:
Awesome spot 💚 . Will update it right away!